source/api_documentation/transactions_api/advertiser_user.rst

```diff
@@ -25,7 +25,7 @@ The API follows REST conventions. Perform an HTTPS GET to the URL with the forma
     - Returns a JSON array of transaction objects. ``https://mynetwork.invoca.net/api/@@TRANSACTION_API_VERSION/advertisers/transactions/33.json``
 
 
-.. include:: intro.rst
+.. include:: _intro.rst
```

Comment:
Thanks for cleaning this up. We only had a convention that custom templates would begin with an underscore but nothing about including other files ("partials" if you will).
Partials don't even have to be .rst files btw, but it looks good to be consistent.